### ISSUE TITLE
Monero + PIN

### ISSUE THREAD

**Issue by l00pz123** [Coinomi/coinomi-android#180]
is there any technical reason why Monero + Ripple are not on the list?
Another issue is PIN that keeps coming up and never was introduced (over a year now)

**Comment by GeorgeKimionis** [comment]
Monero requires a tremendous amount of work on our side as it's unlike any other altcoin we already support and will be available soon: https://twitter.com/CoinomiWallet/status/873710725369393152

As for the PIN feature, it will be introduces in one of our upcoming releases.

**Comment by l00pz123** [comment]
I hope it didn't come out as "gimmie Monero!"
I am genuinely interested in the difficulties of implementations of particular cryptos.
Quality not quantity. Security and reliability above everything else!

**Comment by GeorgeKimionis** [comment]
No, not at all. We wanted to bring Monero on board for a really long time and soon we will be through with the integration. Coinomi is a security-first wallet as you said and we prefer to delay with a release rather than put users' funds at risk, this is why no Coinomi wallet has ever been hacked since 2014 when we released Coinomi's first version.

**Comment by l00pz123** [comment]
And out of curiosity, how are the coins get chosen added to coinomi? 
I noticed there are very rare coins like rubycoin, monacoin, insanecoin and yet no ripple, monero, iota, siacoin, iconomi?

**Comment by sdogeneral** [comment]
Would be really keen to see ripple and tether added, ripple for its liquidity and tether for its so called usd pegging

**Comment by giuseppec** [comment]
Any news regarding Ripple (see https://github.com/Coinomi/coinomi-android/issues/166) ?
